### PR TITLE
Update docs generation, Create new cmake package

### PR DESCRIPTION
**Pull request by nsurbay**
Rewrite frida-qbdi.js to be compatible with sphinx-js

### CODE REVIEW

File: tools/frida-qbdi.js [hunk 1]
```diff
@@ -346,257 +326,247 @@ var QBDI_C = Object.freeze({
...
+//    var GPR_NAMES = ["R0","R1","R2","R3","R4","R5","R6","R7","R8","R9","R10","R12","FP","SP","LR","PC","CPSR"];
+//    var REG_RETURN = "R0";
+//    var REG_PC = "PC";
+//    var REG_SP = "SP";
 } else if (Process.arch === 'ia32'){
     var GPR_NAMES = ["EAX","EBX","ECX","EDX","ESI","EDI","EBP","ESP","EIP","EFLAGS"];
     var REG_RETURN = "EAX";
     var REG_PC = "EIP";
     var REG_SP = "ESP";
 }
```
Comment (bla5r) [inline tools/frida-qbdi.js:355]: We could also add the ARM64 support:

```javascript
if (Process.arch === 'arm64') {
    var GPR_NAMES = ["X0","X1","X2","X3","X4","X5","X6","X7","X8","X9","X10","X11","X12","X13","X14","X15","X16","X17","X18","X19","X20","X21","X22","X23","X24","X25","X26","X27","X28","FP","LR","SP","PC"];
    var REG_RETURN = "X0";
    var REG_PC = "PC";
    var REG_SP = "SP";
}
```

File: tools/frida-qbdi.js [hunk 2]
```diff
@@ -1520,64 +1259,64 @@ function QBDI() {
...
+     */
+    newVMCallback(cbk) {
         if (typeof(cbk) !== 'function' || cbk.length !== 5) {
             return undefined;
         }
         // Use a closure to provide object and a parsed event
         var vm = this;
         var jcbk = function(vmPtr, state, gprPtr, fprPtr, dataPtr) {
-            var s = parseVMState(state);
+            var s = this._parseVMState(state);
```
Comment (bla5r) [inline tools/frida-qbdi.js:1290]: `this` doesn't exist in this scope, call `vm._parseVMState()` instead.

File: tools/frida-qbdi.js (hunk 1 above)
Comment (nsurbay) [reply tools/frida-qbdi.js:355]: QBDI Core doesn't support arm64 yet.

File: docs/source/cmake_target.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
+CMake Integration
+=================
+
+One architecture
+----------------
+
+If you used only one architecture of QBDI in your CMake project, you can import the package ``QBDI`` and ``QBDIPreload``:
```
Comment (romainthomas) [inline docs/source/cmake_target.rst:7]: ~used~ -> use

**Pull request merged**